### PR TITLE
V1.11.0

### PR DESCRIPTION
**Pull request by lina1101**

### CODE REVIEW

File: sokannonser/repository/platsannonser.py [hunk 1]
```diff
@@ -94,6 +94,45 @@ def suggest(args, querybuilder, start_time=0, x_fields=None):
     return result
 
 
+def suggest_extra_word(args, original_word, querybuilder):
+    # PB want it input one word and suggest extra word
```
Comment (erikmelanderAF) [inline sokannonser/repository/platsannonser.py]: Probably should not refer to PB in description of the function.

Comment (lina1101) [reply sokannonser/repository/platsannonser.py]: Delete

File: sokannonser/repository/platsannonser.py [hunk 2]
```diff
@@ -80,20 +80,63 @@ def get_stats_for(taxonomy_type):
...
+                item['value'] = item['value']
+                item['found_phrase'] = item['found_phrase']
     elif args.get(settings.TYPEAHEAD_QUERY):
         result = complete_suggest(args, querybuilder, start_time=0, x_fields=None)
         if not result['aggs']:
             result = phrase_suggest(args, querybuilder, start_time=0, x_fields=None)
     return result
 
 
+def suggest_extra_word(args, original_word, querybuilder):
```
Comment (halge) [inline sokannonser/repository/platsannonser.py]: @lina1101 add description to func

File: sokannonser/repository/platsannonser.py [hunk 3]
```diff
@@ -80,20 +80,63 @@ def get_stats_for(taxonomy_type):
...
+                new_suggest['value'] = search_text + ' ' + result.get('key')
+                new_suggest['found_phrase'] = search_text + ' ' + result.get('key')
+                new_suggest['type'] = search_text_type + '_' + second_suggest_type
+                new_suggest['occurrences'] = result.get('doc_count')
+                new_suggest_list.append(new_suggest)
+    log.debug('List suggest: %s' % new_suggest_list)
+    return new_suggest_list
+
+
+def _check_search_word_type(args, search_text, querybuilder):
```
Comment (halge) [inline sokannonser/repository/platsannonser.py]: @lina1101  same please

Comment (lina1101) [reply sokannonser/repository/platsannonser.py]: second line after this function have the command

File: sokannonser/repository/querybuilder.py [hunk 4]
```diff
@@ -959,5 +959,37 @@ def create_suggest_search(self, suggest, args):
 
         return json.dumps(search)
 
+    def create_check_search_word_type_query(self, word, args):
```
Comment (halge) [inline sokannonser/repository/querybuilder.py]: @lina1101 add func desc

File: sokannonser/repository/querybuilder.py [hunk 5]
```diff
@@ -959,5 +959,37 @@ def create_suggest_search(self, suggest, args):
...
+            aggs['search_type_%s' % field] = {
+                'terms': {
+                    'field': '%s.%s.raw' % (enriched_typeahead_field, field),
+                    'include': word
+                }
+            }
+        return json.dumps(search)
 
-
+    def create_suggest_extra_word_query(self, word, first_word_type, second_word_type, args):
```
Comment (halge) [inline sokannonser/repository/querybuilder.py]: @lina1101 same here

**Pull request merged**